schema/definitions/0.8.0/schema/fmu_meta.json

```diff
@@ -1641,7 +1664,12 @@
           "default": null
         },
         "field_outline": {
-          "$ref": "#/$defs/FieldOutline"
+          "allOf": [
```

Comment:
Why are these now `allOf` with only one schema?

Reply:
Due to some pydantic versions, if v2.5.0 this revertes.